crates/rpc-types/src/eth/transaction/request.rs

```diff
@@ -237,6 +239,28 @@ impl From<Option<Bytes>> for TransactionInput {
     }
 }
 
+impl From<Transaction> for TransactionRequest {
+    fn from(tx: Transaction) -> TransactionRequest {
```

Comment:
could we add a function for this on Transaction as well?
this way we can properly document the limitations?

Reply:
I think we can, but this would have to be `TryFrom` as `TransactionRequest` mostly has `Option`s

Reply:
Or you meant not the reverse conversion function but something like `Transaction::to_request`?

Reply:
yeah, which we then can reuse as the From impl, 
the benefit of this is that there's now a dedicated function we can document